src/components/UsageBar.vue

```diff
@@ -0,0 +1,96 @@
+<template>
+  <div class="header">
+    <slot name="header" />
+  </div>
+  <ProgressBar
+    v-for="(item, index) in items"
+    :badge-label="item.badgeLabel"
+    :key="index"
+    :label="item.label"
+    :max-value="item.maxValue"
+    :value="item.value"
+  />
+  <div class="footer">
+    <slot
+      name="footer"
+      :average-percent="averagePercent"
```

Comment:
What is this used for?

Reply:
Is not in use at the moment, but may be of interest/need to show the average percentage of use in the future.